com.github.geigi.cozy.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "com.github.geigi.cozy",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.geigi.cozy",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-music",
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+     ],
+    "modules": [
+        {
+          "name": "gst-libav",
+          "config-opts": [
+            "--disable-gtk-doc"
+          ],
+          "cleanup": [ "*.la", "/share/gtk-doc" ],
+          "sources": [{
+            "type": "archive",
+            "url": "https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-1.12.3.tar.xz",
+            "sha256": "015ef8cab6f7fb87c8fb42642486423eff3b6e6a6bccdcd6a189f436a3619650"
+          }]
+        },
+        {
+          "name": "gst-plugins-bad",
```

Comment:
Do you need all the plugins here? Overall we prefer to [reduce the scope](https://github.com/flathub/com.github.wwmm.pulseeffects/blob/30ad485fedb0c1d73936911d66f4a078277e0219/com.github.wwmm.pulseeffects.json#L114) as a) less unnecessary storage on the users machine and b) you're affected by fewer potential security issues.

Reply:
Oh somehow this got lost... I've just tested it with both libraries separately and `gst-libav` is enough to support the audio codecs I need (AAC and ALAC for this plugin)